sources/packages/web/src/components/aest/OfferingChangeRequest.vue

```diff
@@ -38,18 +28,10 @@ export default defineComponent({
     const initialData = ref({} as EducationProgramOfferingAPIOutDTO);
 
     onMounted(async () => {
-      let offering: EducationProgramOfferingAPIOutDTO;
-      if (props.relationType === OfferingRelationType.ActualOffering) {
-        offering =
-          await EducationProgramOfferingService.shared.getOfferingDetails(
-            props.offeringId,
-          );
-      } else {
-        offering =
-          await EducationProgramOfferingService.shared.getPrecedingOfferingByActualOfferingId(
-            props.offeringId,
-          );
-      }
```

Comment:
why is this component re-used? Isn't this specific to the offering change request? I was thinking we will be creating a new component/view with `offeringForm.vue` common component. 
![image](https://github.com/bcgov/SIMS/assets/77353155/f2d5922c-f1d0-47c7-a2ed-2842bf4f8d67)

Reply:
Removed this component.